### ISSUE TITLE
Wrong code in moving text

### ISSUE THREAD

**Issue by kidflash** [ChristianChiarulli/Neovim-from-scratch#28]
https://github.com/LunarVim/Neovim-from-scratch/blob/8cebbf6b111b664aecf8a29039c483ae4db5afe7/lua/user/keymaps.lua#L39

The current code is for Insert mode not Normal. I've added the correct code for Insert and Normal mode below:

```
keymap("i", "<A-j>", "<Esc>:m .+1<CR>==gi", opts)
keymap("i", "<A-k>", "<Esc>:m .-2<CR>==gi", opts)
keymap("n", "<A-j>", ":m .+1<CR>==", opts)
keymap("n", "<A-k>", ":m .-2<CR>==", opts)
```

**Comment by aryzing** [comment]
Yeah, noticed this too. Took mine from https://vim.fandom.com/wiki/Moving_lines_up_or_down

**Comment by gnmearacaun** [comment]
fixed by #293
